rockhound/tests/test_bedmap2.py

```diff
@@ -19,6 +20,16 @@ def test_bedmap2_file_name_only():
     assert name.endswith("bedmap2_tiff.zip")
 
 
+def test_bedmap2_datasets_as_str():
+    "Datasets argument passed as strings"
+    for dataset in DATASETS:
+        grid = fetch_bedmap2(dataset)
+        assert set(grid.data_vars) == set([dataset])
+    # Test docstrings="all"
+    grid = fetch_bedmap2("all")
```

Comment:
This seems kind of dangerous since you said that it would eat a lot of memory.

Reply:
Yes. It could be a source of lot of headaches in the future.